File: server/s2game/src/main/java/com/spectopics/dto/LobbyingDto.java
package com.spectopics.dto;

import com.spectopics.s2game.models.LobbyState;
import com.spectopics.s2game.models.Player;

public class LobbyingDto {
    public LobbyState lobbyState;
    public Player player;

    public LobbyingDto(LobbyState lobbyState, Player player) {
        this.lobbyState = lobbyState;
        this.player = player;
    }
}

File: frontend/code/dto/lobbyingDto.py
"""
Docstring for dto.lobbyingDto

helps transfer the data from the server to the client, this will hold lobbyState and the Player
"""

from models.lobbyState import LobbyState
from models.player import Player
from dataclasses import dataclass

@dataclass
class LobbyingDto:
    lobbyState: LobbyState
    player: Player